platform/engineering/design-docs/separate-content-build.md

```diff
@@ -152,56 +152,65 @@ The current front end build will be split up into two distinct builds:
 1. Content build
     - Input: CMS content
     - Output: HTML files, assets from the CMS (PDFs, images and the like)
+    - This maps pretty cleanly to the current partial deploy
 1. Application build
     - Input: `vets-website` code
     - Output: JavaScript and CSS bundles
+    - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to separate buckets. The
-reverse proxies will route traffic intelligently to the appropriate bucket given
-the request.
-
-![Proposed traffic flow to S3 buckets](images/proposed-traffic-flow-to-s3-buckets.png)
-
-**Important note:** JavaScript application landing pages will live in the CMS.
-To get an application into production, the page in Drupal will need to be
-published.
+The output of both these builds will be deployed to a single S3 bucket with
+safeguards in place to ensure they don't override each other.
 
 **Another important note:** There is no way to coordinate these two deploys to
 make an application live for the first time. The process will be to manually:
 1. Verify the application assets are live in production
-1. Publish the landing page
+1. Make the landing page live in production in the content build
+    - Depending on the answer to [this
+      question](#react-application-landing-pages), this may mean publishing the
+      page in Drupal or switching the flag in the JSON file
 1. Deploy a new content build
 
 ## Specifics
 
 ### Detailed Design
 
-#### Build and deploy
-- The content build will live in a separate repository
-  - **Question:** What should we call this?
-  - **Question:** Can the application build be run in Circle?
-  - **Question:** Where can we run the content build?
-- Content will be deployed to a separate S3 bucket from the application code
-- The S3 buckets will be [versioned](https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html)
-  in them with the contents of each deployment
-  - Old directories will be removed after 90 days
-    - **Question:** What makes sense? This is a totally arbitrary number
-
-#### Routing
-- The nginx will route their traffic based on some configuration file
-  - **Question:** What configuration file? How does it work?
+#### Content build in a separate repository
+The content build will live in a separate repository called `content-build`.
+This repository will contain only code pertaining to the content build.
+
+![Separated content code base](images/phase-2-separated-content-build.png)
+
+#### Coordinating deployments to S3
+We use S3 to serve a static site. To do this, the bucket consists of a file tree
```

Comment:
For my own knowledge: what handles this translation of "requested URL" to "bucket to fetch file from"?

Reply:
I believe this is the forward proxy? cc @nathanhruby

Reply:
Also, to be clear, after talking with Nathan, I changed the approach from multiple buckets to a single bucket, so this is the same as it is now.

Reply:
Just to clean up the loose ends, I'm marking this as resolved for now. Feel free to reopen if needed, though.